app/graphql/types/mutation_type.rb

```diff
@@ -1,4 +1,16 @@
 class Types::MutationType < Types::BaseObject
+  # rubocop:disable Layout/LineLength
+
   # Kitsu Pro
-  field :pro, Types::ProMutation, null: false
+  field :subscribe_with_stripe, mutation: Mutations::Pro::SubscribeWithStripe, description: 'Subscribe to Pro using Stripe'
+  field :unsubscribe_pro, mutation: Mutations::Pro::Unsubscribe, description: "End the user's pro subscription"
+  field :set_pro_message, mutation: Mutations::Pro::SetMessage, description: "Set the user's Hall-of-Fame message"
+  field :set_pro_discord, mutation: Mutations::Pro::SetDiscord, description: "Set the user's discord tag"
+
```

Comment:
Is there a reason for flattening this?  I know it's unorthodox to have a tree of mutations but I really cannot fathom having a list for the literal thousands of mutations in our app

AFAIK the only downside is unspecified execution order for multiple mutations in the same query, which I'm not convinced is a major issue?

Reply:
Yeah I think we can work around the execution order thing. But I'm not a big fan of how it looks to write the graphql for this.

eg.
```
mutation UpdateAnime($id: ID!, $input: UpdateAnimeInput!) {
  anime {
	updateAnime(id: $id, input: $input) {
      <fields>
    }
	<whatever other anime mutations>
  }
}
```

It's just a couple extra lines but it feels bad to add more boilerplate where we don't need to. And then you have to dive another level to get at the result when it comes back. And it's a slippery slope towards multiple levels of nesting. I'm of the mind we shouldnt sacrifice the api to make our file structure better. But could we maybe like just reopen the root mutation type in multiple files to keep it organized that way so we have the best of both worlds?

Reply:
My bigger worry is how this shows up in api docs, being a giant flat list of RPC with no namespacing

Reply:
Ah. Yeah i mean we could standardize on having the "namespace" be at the start of the mutation name. But i don't feel too strongly about it either way. With the non-flat structure we could also do `anime { update() }` instead of `anime { updateAnime() }` or `anime { animeUpdate() }`.